Source/Core/Core/PowerPC/Jit64/Jit_LoadStorePaired.cpp

```diff
@@ -32,8 +32,12 @@ void Jit64::psq_stXX(UGeckoInstruction inst)
 	int w = indexed ? inst.Wx : inst.W;
 	FALLBACK_IF(!a);
 
+	std::map<u8, u32>::iterator it = js.constantGqr.find(i);
+	bool gqrIsConstant = it != js.constantGqr.end();
+	u32 gqrConstant = it->second & 0xffff;
```

Comment:
Is it safe to dereference "it" if no constant value is found?
Maybe: u32 gqrConstant = gqrIsConstant ? it->second & 0xffff : 0;